Runtime/Interaction/StaticMeshCollidersManager.cs

```diff
@@ -0,0 +1,213 @@
+            
+            if (vertexRemapData == null)
+            {
+                throw new ArgumentException("Missing Vertex Remap Data Asset! Create a new one or provide an existing one!");
+            }
+            
+            tempMesh.GetVertices(vertices);
+            tempMesh.GetNormals(normals);
+            
+            remapped_vertices_data = new NativeArray<int>(vertexRemapData.RemappedVertices, Allocator.Persistent);
+            
+            meshXRes = hpuiStaticContinuousInteractable.MeshXResolution;
+            meshYRes = vertices.Count / meshXRes;
+            float xWidth = Vector3.Distance(vertices[remapped_vertices_data[0]], vertices[remapped_vertices_data[1]]);
+            float yWidth = Vector3.Distance(vertices[remapped_vertices_data[0]], vertices[remapped_vertices_data[meshXRes]]);
+            float offsetX = xWidth * meshXRes * 0.5f;
+            float offsetY = yWidth * meshYRes * 0.5f;
+            Transform[] colliderTransforms = new Transform[vertices.Count];
+            Transform meshTransform = targetMesh.gameObject.transform;
+            InitCollidersState(meshTransform, xWidth, yWidth, colliderTransforms, offsetX, offsetY);
+            colliderObjects = new TransformAccessArray(colliderTransforms);
+            vertices_native = new NativeArray<Vector3>(vertices.ToArray(), Allocator.Persistent);
+            normals_native = new NativeArray<Vector3>(normals.ToArray(), Allocator.Persistent);
+            generatedColliders = true;
+
+            return colliderCoords.Keys.ToList();
+        }
+
+        private void InitCollidersState(Transform meshTransform, float xWidth, float yWidth, Transform[] colliderTransforms
+            , float offsetX, float offsetY)
+        {
+            for (int i = 0; i < remapped_vertices_data.Length; i++)
+            {
+                int x = i % meshXRes;
+                int y = i / meshXRes;
+                GameObject colliderGameObject = new GameObject();
+                Collider col = colliderGameObject.AddComponent<BoxCollider>();
+                colliderGameObject.name = "X: " + x + "; Y: " + y + ";";
+                colliderGameObject.transform.parent = meshTransform;
+                colliderGameObject.transform.localPosition = vertices[remapped_vertices_data[i]];
+                Vector3 targetScale = Vector3.one * scaleFactor;
+                targetScale.x = xWidth;
+                targetScale.y = yWidth;
+                colliderGameObject.transform.localScale = targetScale;
+                colliderGameObject.transform.localRotation = Quaternion.LookRotation(normals[remapped_vertices_data[i]]);
+                colliderTransforms[i] = colliderGameObject.transform;
+                Vector2 coords = Vector2.zero;
+                if (flippedAndNormalisedCoordinates)
+                {
+                    coords = new Vector2((float)((meshXRes - 1) - x)/meshXRes, (float)((meshYRes - 1) - y)/meshYRes);
```

Comment:
fyi, this breaks the assumption of the API. I shouldn't return normalized values. This is to make sure other interactables also can understand what this means

Reply:
fixed (I think?)

Reply:
`ComputeInteractorPosition` is expected to return values as unity units. Now, when that flag is set, it returns something else. If you want to sort that out, probably have the Interactable return the grid size & resolution and compute that outside of the HPUI-Core. For example, the HPUIContinuousInteractable exposes those parameters to be used if necessary.

Reply:
fixed